### ISSUE TITLE
Trailing spaces in queryads

### ISSUE THREAD

**Issue by XhmikosR** [pi-hole/AdminLTE#1259]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [ ] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/AdminLTE/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/AdminLTE/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1, although it seems it's coming from JS https://github.com/pi-hole/AdminLTE/blob/b86e4a3178e46cef9fc738c6a0fb8d0ef0ff2fbc/scripts/pi-hole/js/queryads.js#L12-L22

---

```console
user@foo:~ $ pihole -v
  Pi-hole version is v5.0 (Latest: v5.0)
  AdminLTE version is v5.0 (Latest: v5.0)
  FTL version is v5.0 (Latest: v5.0)
```

This is mostly nitpicking :) So, currently, the queryads output includes extra whitespace at the end and one before all lines. 

```diff
diff --git "a/C:\\Users\\XhmikosR\\Desktop\\1.txt" "b/C:\\Users\\XhmikosR\\Desktop\\2.txt"
index ae40516..ba35bd1 100644
--- "a/C:\\Users\\XhmikosR\\Desktop\\1.txt"
+++ "b/C:\\Users\\XhmikosR\\Desktop\\2.txt"
@@ -1,11 +1,11 @@
- Match found in https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts:
-   ssl.google-analytics.com 
-   www.google-analytics.com 
-   google-analytics.com 
- Match found in https://www.github.developerdan.com/hosts/lists/ads-and-tracking-extended.txt:
-   google-analytics.com 
-   cctldtest.google-analytics.com 
-   click.google-analytics.com 
-   ssl.google-analytics.com 
-   www.google-analytics.com 
-   ssl.google-analytics.comstatic.xx.fbcdn.nett.skimresources.com 
\ No newline at end of file
+Match found in https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts:
+  ssl.google-analytics.com
+  www.google-analytics.com
+  google-analytics.com
+Match found in https://www.github.developerdan.com/hosts/lists/ads-and-tracking-extended.txt:
+  google-analytics.com
+  cctldtest.google-analytics.com
+  click.google-analytics.com
+  ssl.google-analytics.com
+  www.google-analytics.com
+  ssl.google-analytics.comstatic.xx.fbcdn.nett.skimresources.com
\ No newline at end of file

```

**Comment by yubiuser** [comment]
Additional white spaces are still there

**Comment by rdwebdesign** [comment]
Fixed by #2079
